### ISSUE TITLE
Can you this?

### ISSUE THREAD

**Issue by LearnMiner** [PipeDeveloper/XmrigCC-MOD#5]
Detect if system is 32 or 64 for execute xmrig 32 or 64 for better hashrate
when user is idle mine 100% cpu
when user is idle miner gpu too

**Comment by Zelecktor** [comment]
IDLE you mean start mining with 100% cpu when PC is idle and 50% when PC is in use. Something like that?

**Comment by LearnMiner** [comment]
yes mining all time 50% then when user is idle mine with 100% cpu
